### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by TiagoLucas22478**
Following the update of the SECURITY.md template, please review the filled-in information

### CODE REVIEW

**Comment by mrybczyn** [comment]
This update (with Project's touch if needed) is recommended because: 

Reporting of security issues by default moved to GitLab. Bugzilla is closed for security reports
The Security team webpage was redesigned a bit. There are redirects, but it is a good idea to use the correct links
The current email for the security team is security@eclipse-foundation.org

File: SECURITY.md [hunk 1]
```diff
@@ -1,49 +1,25 @@
...
-## Reporting a Security Vulnerability
+# Supported Versions
 
-Vulnerabilities can be reported either via email to the Eclipse Security Team
-or directly with a project via the Eclipse Foundation's Bugzilla instance.
+<!--
+    Which releases of the project's software are actively maintaned and receive security updates?
+-->
+Supported versions are:
+* Jakarta EE 10
```
Comment (ivargrimstad) [inline SECURITY.md]: This is not correct. EE4J is a GH org mostly for implementations, some of which are not tied directly to any Jakarta EE release. The Jakarta EE related repositories have moved, or is in the process of moving to the Jakarta EE GH org.

**Review by ivargrimstad — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (TiagoLucas22478) [reply SECURITY.md]: Pardon my ignorance, but what should be in there instead?
@mrybczyn suggested we can also not include this section on this repo but have a separate SECURITY.md for each project repo

Comment (ivargrimstad) [reply SECURITY.md]: I think it should be left empty. The EE4J org includes repositories for more than 20 different products, all with different release cycles and versions.

Comment (ivargrimstad) [reply SECURITY.md]: Maybe something like: "Check the individual repositories for latest versions."

**Review by ivargrimstad — APPROVED** [review]

**Pull request merged**